File: app/src/main/java/com/tarento/checkin/checkin/RegistrationActivity.java
package com.tarento.checkin.checkin;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class RegistrationActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_registration);
    }

    public void register(View view) {
        Intent intent = new Intent(this, SuccessfulRegistrationActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/tarento/checkin/checkin/RegistrationActivity.kt
package com.tarento.checkin.checkin

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class RegistrationActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_registration)
    }

    fun register(view: View) {
        val intent = Intent(this, SuccessfulRegistrationActivity::class.java)
        startActivity(intent)
        finish()
    }

    fun capturePhoto(view: View) {
        val intent = Intent(this, CameraActivity::class.java)
        startActivity(intent)
    }
}
